CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing to BETER Feed Testing Sandbox (BFTS)
+
+## Introduction
+
+Thank you for considering contributing to the BETER Feed Testing Sandbox (BFTS) project! This document provides guidelines and instructions to help you contribute effectively.
+
+## Table of Contents
+
+1. [Code of Conduct](#code-of-conduct)
+2. [How to Contribute](#how-to-contribute)
+   - [Fork the Repository](#fork-the-repository)
+   - [Create a Branch](#create-a-branch)
+   - [Make Changes](#make-changes)
+   - [Test Your Changes](#test-your-changes)
+   - [Submit a Pull Request](#submit-a-pull-request)
+3. [Guidelines for Development](#guidelines-for-development)
+   - [Resource Requirements](#resource-requirements)
+   - [Load testing resources requirements](#load-testing-resources-requirements)
+   - [Versioning Process](#versioning-process)
+4. [Reporting Issues](#reporting-issues)
+5. [Getting Help](#getting-help)
+
+## Code of Conduct
+
+This project adheres to a [Code of Conduct](CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [email@example.com](mailto:email@example.com).
```

Comment:
There is no CODE_OF_CONDUCT + email is invalid.
I have no idea where you got it, but I suggest removing the whole section Code of Conduct. PRs are the proper way to report cases.